### ISSUE TITLE
"Previously was" is wrongly synchronized every time a change is made

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/notes#239]
**[Notes]:**
- This issue is reproducible only if both instances are opened at the same time.
- If the instances are edited multiple times, the content becomes illegible.

**[Affected versions]:**
- Firefox Notes 1.8.0dev
- Firefox 55 and up

**[Affected Platforms]:**
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have two Firefox profiles with Firefox Notes 1.8.0dev installed.
- You must be logged with the same sync account on both profiles.

**[Steps to reproduce]:**
1. Open the browser with the profiles from prerequisites.
2. Write something in the first instance and then in the second.
3. Change or add content in the second instance.
4. Change or add content in the first instance.
5. Repeat steps 4 and 3.
6. Observe both instances.

**[Expected result]:**
- "Previously was" text is displayed only once.

**[Actual result]:**
- The "Previously was" text is multiplied.

**[Additional Notes]:**
- Attached a screen recording of the issue:
![previously was](https://user-images.githubusercontent.com/31243259/30601530-e4c3933c-9d6a-11e7-8578-19a1de496d1a.gif)

**Comment by johngruen** [comment]
Closing in favor of #238, but may have more detail
